drivers/spi/spi_mcux_lpspi.c

```diff
@@ -130,8 +130,12 @@ static void spi_mcux_transfer_next_packet(const struct device *dev)
 	status = LPSPI_MasterTransferNonBlocking(base, &data->handle,
 						 &transfer);
 	if (status != kStatus_Success) {
-		LOG_ERR("Transfer could not start");
+		LOG_ERR("Transfer could not start on %s: %d",
+			dev->name, status);
 	}
```

Comment:
Minor nit here- why not add the code that handles the error return case within the `if` block?

Reply:
Thanks @danieldegrasse. It should now be fixed.